Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -487,7 +488,8 @@ def format_results(client, uuid):
         human_readable['Screenshot'] = scan_tasks['screenshotURL']
         screen_path = scan_tasks['screenshotURL']
         response_img = requests.request("GET", screen_path, verify=client.use_ssl)
-        stored_img = fileResult('screenshot.png', response_img.content)
+        screenshot_name = cont['EffectiveURL'].replace('http://', '').replace('https://', '').replace('/', '_')
+        stored_img = fileResult('{}.png'.format(screenshot_name), response_img.content)
```

Comment:
This may be a long string causing the file name to exceed MAX_PATH length.

Also, This is a breaking change.
Add a new argument called: `use_url_as_name` (default: false).

Reply:
added `use_url_as_name` argument: https://github.com/demisto/content/pull/21272/commits/b22484a5ee09665876fccc386f839fbd94e76abc

Reply:
How do we handle if the string is too long?